File: 4-1/Android-java/DragonWar/app/src/main/java/com/example/dragonwar/framework/MainActivity.java
package com.example.dragonwar.framework;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import com.example.dragonwar.framework.GameView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(new GameView(this));
    }
}

File: 4-summer/kotlin/gpsdamagochi/app/src/main/java/com/enjcat/gpsdamagochi/MainActivity.kt
package com.enjcat.gpsdamagochi

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        var bools = true

        button3.setOnClickListener {
            if(bools == true) {
                textView.text = "버튼이 눌렸습니다."
                bools = false
            }
            else{
                textView.text = getString(R.string.hello_world)
                bools = true
            }
        }
    }
}
